internal/cmd/volume/create.go

```diff
@@ -35,7 +35,7 @@ var CreateCommand = base.Cmd{
 		cmd.MarkFlagRequired("size")
 
 		cmd.Flags().Bool("automount", false, "Automount volume after attach (server must be provided)")
-		cmd.Flags().String("format", "", "Format volume after creation (automount must be enabled)")
+		cmd.Flags().String("format", "", "Format volume after creation")
```

Comment:
Regarding your TODO, I think we can just hardcode them, we did something similar for the `network add-subnet --type` flag. If we ever add another format, the user can still pass it in because we dont validate against the autocompletion.

```suggestion

		cmd.Flags().String("format", "", "Format volume after creation (ext4 or xfs)")
		cmd.RegisterFlagCompletionFunc("format", cmpl.SuggestCandidates("ext4", "xfs"))
```